staging/src/k8s.io/client-go/tools/auth/clientauth.go

```diff
@@ -87,6 +86,7 @@ type Info struct {
 // If the file does not exist, then os.IsNotExist(err) == true
 func LoadFromFile(path string) (*Info, error) {
 	var info Info
+	// TODO: @perithompson we should log these steps because this function can hang in congested windows clusters
```

Comment:
revert the change from this file in favor of an issue?

Reply:
done